src/renderer/components/settings/field/password.vue

```diff
@@ -1,19 +1,20 @@
 <template>
-    <div class="grid-x grid-padding-x">
-      <label class="cell" :for="field.name+'-id'">
-        {{ field.display || field.name }}
-      </label>
-      <div class="cell small-4 flex-container">
-        <input
-            :type="inputType"
-            :name="field.name"
-            :id="field.name+'-id'"
-            :minLength="field.minLength"
-            :maxLength="field.maxLength"
-            @change="$emit('value-change', $event.target.value)"
-        />
-        <button class="secondary small button" @click="toggleShow"><i class="fas" v-bind:class="{ 'fa-eye-slash': isShowingPassword, 'fa-eye': !isShowingPassword }"></i></button>
-      </div>
+    <div class="inline fields">
```

Comment:
This would look better as an [`action input`](https://semantic-ui.com/elements/input.html#action).